docs/noencap-hybrid-modes.md

```diff
@@ -0,0 +1,174 @@
+    #                    SNAT'd if noSNAT is not set to true. Underlying network must be capable of
+    #                    supporting Pod traffic across IP subnets.
+    # hybrid:            noEncap if source and destination Nodes are on the same subnet, otherwise encap.
+    # networkPolicyOnly: Antrea enforces NetworkPolicy only, and utilizes CNI chaining and delegates Pod
+    #                    IPAM and connectivity to the primary CNI.
+    #
+    trafficEncapMode: noEncap
+
+    # Whether or not to SNAT (using the Node IP) the egress traffic from a Pod to the external network.
+    # This option is for the noEncap traffic mode only, and the default value is false. In the noEncap
+    # mode, if the cluster's Pod CIDR is reachable from the external network, then the Pod traffic to
+    # the external network needs not be SNAT'd. In the networkPolicyOnly mode, antrea-agent never
+    # performs SNAT and this option will be ignored; for other modes it must be set to false.
+    #noSNAT: false
+    ... ...
+```
+
+After changing the options, you can deploy Antrea with the `noEncap` mode by
+applying the deployment YAML.
+
+### Using kube-router for BGP
+
+We can run kube-router in the advertisement-only mode to advertise Pod CIDRs to
+the peered routers, so the routers can know how to route Pod traffic to the
+Nodes. To deploy kube-router in the advertisement-only, first download the
+[kube-router DaemonSet template](https://raw.githubusercontent.com/cloudnativelabs/kube-router/v0.4.0/daemonset/generic-kuberouter-only-advertise-routes.yaml):
+
+```bash
+curl -LO https://raw.githubusercontent.com/cloudnativelabs/kube-router/v0.4.0/daemonset/generic-kuberouter-only-advertise-routes.yaml
+```
+
+Then edit the YAML file and set the following kube-router arguments:
+
+```yaml
+- "--run-router=true"
+- "--run-firewall=false"
+- "--run-service-proxy=false"
+- "--enable-cni=false"
+- "--enable-ibgp=false"
+- "--enable-overlay=false"
+- "--enable-pod-egress=false"
+- "--peer-router-ips=<CHANGE ME>"
+- "--peer-router-asns=<CHANGE ME>"
+```
+
+The BGP peers should be configured by specifying the --peer-router-asns and
+--peer-router-ips parameters. Note, the ASNs and IPs must match the
+configuration on the peered routers. For example:
+
+```yaml
+- "--peer-router-ips=192.168.1.99,192.168.1.100
+- "--peer-router-asns=65000,65000"
+```
+
+Then you can deploy the kube-router DaemonSet by:
+
+```bash
+kubectl apply -f generic-kuberouter-only-advertise-routes.yaml
+```
+
+You can verify the kube-router Pods are running on the Nodes of your Kubernetes
+cluster by (the cluster in the following example has only two Nodes):
+
+```bash
+$ kubectl -n kube-system get pods -l k8s-app=kube-router
+NAME                READY     STATUS    RESTARTS   AGE
+kube-router-rn4xc   1/1       Running   0          1m
+kube-router-vhrf5   1/1       Running   0          1m
+```
+
+Antrea can be deployed either before or after kube-router, with the `NoEncap`
+mode.
```

Comment:
s/with the `NoEncap` mode/in `NoEncap` mode

Reply:
Done